app/src/main/java/ch/epfl/polybazaar/Utilities.java

```diff
@@ -64,6 +64,14 @@ public static boolean emailIsValid(String email) {
         return (email.matches("[a-zA-Z]+"+"."+"[a-zA-Z]+"+"@epfl.ch"));
     }
 
+    public static boolean passwordIsValid(String password) {
+        return password.length() >= 6;
```

Comment:
I would store the 6 as a constant instead of a magic number. Also if you choose a nickname shorter than 6 it only tells you that it is not valid (but not why it is not valid)